alts/shared/models.py

```diff
@@ -205,7 +213,7 @@ def __init__(self, **data):
         for (
             field_name,
             field,
-        ) in self.results_backend_config.dict().items():
+        ) in self.results_backend_config.model_fields.items():
```

Comment:
Use `model_dump` here, we need a dictionary of the data, not the model fields information